game.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<!--PAGE SETUP-->
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Whack-a-Rat</title>
+    <meta name="description" content="Mayden Academy JS Game">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/game.css">
+</head>
+  <body>
+    <!--SPLASH SCREEN-->
+    <div class="splash">
+        <div class="gamescreen">
+            <img class="image" src="assets/imgs/exploding.svg">
+
+            <div class="buttons">
+                <h1 class="heading">whack-a-rat!</h1>
+                <button class="button start_button">start</button>
+                <button class="button instruction_button">instructions</button>
+            </div>
+        </div>
+    </div>
+
+    <!--INSTRUCTIONS-->
+        <div class="instructions-container no_show">
+            <h1>Whack-A-RAT</h1>
+            <h2>Game Instructions:</h2>
+            <br>
+            <p>1. See the rat.<br>
+                2. Whack the rat.<br>
+                3. You're done.<br>
+                ...<br><br>
+                Actually, there's something more than that, so let's try again :)<br>
+                1. Hit the START button on the menu screen.<br>
+                2. Now you are on a game screen, which looks like a comfy class room, probably the one from Mayden Academy. There are 6 pretty desks. Rats will appear from behind the desks.<br>
+                3. To let you get ready for hitting rats, the countdown timer will count 3...2...1...GO!<br>
+                4. Once a countdown is finished, rats will start randomly appear on their desks.<br>
+                5. Make rats disappear simply clicking on any part of them. Even whiskers are in the game >.<<br>
+                6. All rats you miss will stay for 1 second and then disappear themselves.<br>
+                7. Each rat you hit will bring you 1 point.<br>
+                8. You could see your score in a top right corner of a game screen. Don't look there though, concentrate on rats, because...<br>
+                9. Every 5 seconds the game will become more challenging and more rats will appear, try your best to whack them all, AND....<br>
+                10. Remember, you will only have 30 seconds to become a Whack-A-RAT world champion. Once 30 seconds are up, the game will stop and you will see your impressive final score and success rate.<br>
+                Thank you for reading.<br>
+                Now, let's whack some rats !</p>
+            <button class="back">Go Back!</button>
```

Comment:
the button and text dont actually fit in the screen, this looks terrible. please CSS better